src/Robin/Ntlm/Crypt/Random/OpenSslRandomByteGenerator.php

```diff
@@ -15,7 +15,7 @@
  * "openssl" extension.
  *
  * @link http://php.net/openssl
- * @deprectated NOTE! This implementation is deprecated, as it's been found to
+ * @deprecated Insecure! This implementation is deprecated, as it's been found to
```

Comment:
Also is this an example task tag? The deprecation itself is that. This is more FYI I think.